plugins/statsd/README.md

```diff
@@ -0,0 +1,86 @@
+# Statsd plugin for DC/OS Metrics
+
+This plugin will forward all data received by dcos-metrics to a statsd
+server, as defined by STATSD_HOST and STATSD_PORT environment variables.
+
+## Architecture
+
+The statsd plugin uses the dcos-metrics plugin API. You run the 'plugin' (a
+slight misnomer; it is in fact a separate binary) on each node in your cluster.
+It polls its local HTTP API for metrics, which it re-transmits to your statsd
+server.
+
+### Limitations
+
+ - statsd has no units, therefore *units are not sent*
+ - statsd has no tags, therefore *dimensions are not sent*
```

Comment:
If we are discarding dimensions does that mean that metrics will be overwritten in statsd, or does it mean that metrics will all be aggregates?

Reply:
The metrics are gauges, so the most recent one will overwrite the previous one. The effect is that identical metrics from multiple sources are not useful. This is a statsd limitation.